stage1/init/init.go

```diff
@@ -122,8 +122,18 @@ func getArgsEnv(p *Pod, debug bool) ([]string, []string, error) {
 
 	case "src":
 		args = append(args, filepath.Join(common.Stage1RootfsPath(p.Root), nspawnBin))
-		args = append(args, "--boot") // Launch systemd in the pod
-
+		args = append(args, "--boot")              // Launch systemd in the pod
+		args = append(args, "--register", "false") // We cannot assume the host system is running systemd
+		out, err := os.Getwd()
+		if err != nil {
+			return nil, nil, err
+		}
+		lfd, err := common.GetRktLockFD()
+		if err != nil {
+			return nil, nil, err
+		}
+		args = append(args, fmt.Sprintf("--pid-file=%v", filepath.Join(out, "pid")))
```

Comment:
Since v219, systemd-nspawn has a [patch to pass the container pid](http://cgit.freedesktop.org/systemd/systemd/commit/?id=733d15ac7a23c80f2e447f6c2fca0406bc9960db). This would be more complex to use but we could reference this in a comment.